### ISSUE TITLE
Refactor large exports to use deferred downloading

### ISSUE THREAD

**Issue by joshsadam** [phac-nml/irida#1367]
## What needs changed?

Once the request has been received by the server, track the request (session state?),  Once the download is ready for actual downloading notify the user so they can download it.  This will probably require web sockets.

## How does it work now?

Currently, there is an issue (reported by Jennifer Tanner on Sept 15, 2022) that when downloading a group of large sequence files it the download errors out.  This is do to spring's current asynchronous timeout.  Adjusting this was done as a temporary fix.

## How should it work?

How do you think this component should work?

## Additional information

Add any other context, files, or screenshots about this request here.
